lib/js/components/ProgressBar/ProgressBar.consts.ts

```diff
@@ -0,0 +1,41 @@
+export const PROGRESS_BAR_COLORS = {
+	NEUTRAL: 'neutral',
+	INFO: 'info',
+	PRIMARY: 'primary',
+	SUCCESS: 'success',
+	WARNING: 'warning',
+	FAIL: 'fail',
+} as const;
+
+export const PROGRESS_BAR_SIZES = {
+	MEDIUM: 'medium',
+	SMALL: 'small',
+	XSMALL: 'extra small',
+} as const;
+
+export const PROGRESS_BAR_RADII = {
+	DEFAULT: 'default',
+	NONE: 'none',
+};
+
+export const PROGRESS_BAR_LAYOUTS = {
+	DEFAULT: 'default',
+	COMPACT: 'compact',
+};
+
+export const PROGRESS_BAR_LAYERS = {
+	ONE: 1,
+	TWO: 2,
+};
+
+export const PROGRESS_BAR_COLOR_SCHEMES = {
+	DEFAULT: 'default',
+	MEDIUM: 'medium',
+	MEDIUM_NEUTRAL: 'medium-neutral',
+};
+
+export interface Ranges {
```

Comment:
this type defines one range. Why did you use plural in name?

Reply:
Fixed.